crates/solver/src/settlement_rater.rs

```diff
@@ -19,14 +19,8 @@ use web3::types::AccessList;
 type SolverSettlement = (Arc<dyn Solver>, Settlement);
 pub type RatedSolverSettlement = (Arc<dyn Solver>, RatedSettlement, Option<AccessList>);
 
-pub struct SimulationDetails {
-    pub settlement: Settlement,
-    pub solver: Arc<dyn Solver>,
-    /// Which storage the settlement tries to access. Contains `None` if some error happened while
-    /// estimating the access list.
-    pub access_list: Option<AccessList>,
-    /// The outcome of the simulation. Contains either how much gas the settlement used or the
-    /// reason why the transaction reverted during the simulation.
```

Comment:
IMO without this comment it's not obvious how the field is to be understood.

Reply:
Reverted.